code/datums/components/rotation.dm

```diff
@@ -131,21 +130,21 @@
 			return FALSE
 		var/obj/item/tool = user.get_active_held_item()
 		if(!tool || tool.tool_behaviour != TOOL_WRENCH)
-			rotated_obj.balloon_alert(user, "need a wrench")
+			rotated_obj.balloon_alert(user, "need a wrench!")
 			return FALSE
 	if(!(rotation_flags & ROTATION_IGNORE_ANCHORED) && rotated_obj.anchored)
 		if(istype(rotated_obj, /obj/structure/window))
-			rotated_obj.balloon_alert(user, "need to unscrew")
+			rotated_obj.balloon_alert(user, "need to unscrew!")
 		else
-			rotated_obj.balloon_alert(user, "need to unwrench")
+			rotated_obj.balloon_alert(user, "need to unwrench!")
```

Comment:
Honestly, should probably just get rid of these alerts for the anchor check. Stuff like plumbing machines and hydroponics trays use `alt click` for other things, and simple rotation will constantly pop up alerts that you need to unanchor it even if you're not trying to

The unwrench balloon alert is probably fine, though

Reply:
Every time I alt-click a hydroponics tray to empty its chemicals, I get a 'need to unwrench' popup, so.

Reply:
This balloon alert is useful, I think those things just need to be able to include ways to tell the signal to not show up at certain points, like with a callback (cancelling wouldn't work because this is assuming they're all on the same alt-click signal)

Reply:
I'm going to move any procs that use AltClick to another hotkey. (most likely RMB) We use CtrlClick to drag things as the standard that has been set in stone for a long time.  The same precedence applies to rotation and AltClick.  Both of these should always be uniform across all objects. (with rare exceptions)